packages/core/src/external/commonwell/document/document-downloader-local.ts

```diff
@@ -245,4 +272,22 @@ export class DocumentDownloaderLocal extends DocumentDownloader {
       throw new MetriportError(`CW - Error downloading document`, error, additionalInfo);
     }
   }
+
+  protected async updateContentTypeInS3(
```

Comment:
currently, we dont update the file extension, just the content type. So a .tiff file will have a .bin extension. We could update extension too but would require a copy and delete operation like this 

`protected async updateContentTypeInS3(bucket: string, key: string, newContentType: string, newExtension: string): Promise<void> {
  const newKey = key.replace(/\.[^/.]+$/, `.${newExtension}`);
  const copySource = encodeURIComponent(bucket + "/" + key);

  // Copy the object to the new key
  await this.s3client.copyObject({
    Bucket: bucket,
    Key: newKey,
    CopySource: copySource,
    ContentType: newContentType,
    MetadataDirective: 'REPLACE' // This is important to replace the metadata
  }).promise();

  // Delete the old object
  await this.s3client.deleteObject({
    Bucket: bucket,
    Key: key
  }).promise();
}`

Reply:
I suggest we rename the file accordingly.